public/playpass-content.schema.json

```diff
@@ -1,15 +1,16 @@
 {
   "type": "object",
   "properties": {
```

Comment:
please make sure the optional field for musicVideoLink is included in the schema.  It's very important on youtube that we can source audio from two different files, since the user reward is better when it's a music video like a live performance while the game itself should be an album version of the song.